### PR TITLE
added DAO layer for Order,ShoppingCart,User and creator id for them

### PR DESCRIPTION
**Pull request by matviichuk92**

### CODE REVIEW

**Comment by jah701** [comment]
Add Travis

**Review by jah701 — APPROVED** [review]
Looks nice for me

File: src/main/java/online/shop/db/Storage.java [hunk 1]
```diff
@@ -2,15 +2,42 @@
...
 
     public static void addProduct(Product product) {
         productId++;
         product.setId(productId);
         products.add(product);
     }
+
+    public static void addUser(User user) {
+        userId++;
+        user.setId(userId);
```
Comment (RomanDubovskyi) [inline src/main/java/online/shop/db/Storage.java]:
```suggestion
        user.setId(++userId);
```
same with other methods

File: src/main/java/online/shop/service/impl/OrderServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+public class OrderServiceImpl implements OrderService {
+    @Inject
+    private OrderDao orderDao;
+
+    @Inject
+    private ShoppingCartService shoppingCartService;
+
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        Order order = new Order(shoppingCart.getId());
```
Comment (RomanDubovskyi) [inline src/main/java/online/shop/service/impl/OrderServiceImpl.java]:
```suggestion
        Order order = new Order(shoppingCart.getUserId());
```

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**